### DISCUSSION TITLE
[Feature] Ability to set an individual playback speed for Video(s)-on-Demand

### DISCUSSION THREAD

**Discussion by Buedenbender** [community/events#49] [Keynotes]
Thanks for this wonderful conference, which is going so well this far.
I am quite sure you are busy with the organization, and  ... I can not judge how 
difficult my suggestion would be to implement, but I would love to have
the ability to set an individual playback speed on the videos on demand.

Just as an idea.
Also, I did not see it yet, how long will the videos on demand be available to us?

**Comment by Bestra** [comment]
This isn't official but from what I can see all the recorded stuff from Universe 2021 [is on YouTube](https://www.youtube.com/watch?v=8JjVNFc2kK4&list=PL0lo9MOBetEFwWAsfaW3BZ9-NV_C_cdw3), and I'm betting it'll be that way for 2022. That doesn't fix the playback speed in the short term, but YouTube also has variable playback speed.

**Comment by jricciardi** [comment]
Re-Categorizing this to `Keynotes` so it gets a little more visibility 😄
